shell/browser/ui/electron_web_modal_dialog_manager_delegate.cc

```diff
@@ -0,0 +1,15 @@
+#include "shell/browser/ui/electron_web_modal_dialog_manager_delegate.h"
+
+#include "content/public/browser/web_contents.h"
+#include "shell/common/platform_util.h"
+
+ElectronWebModalDialogManagerDelegate::ElectronWebModalDialogManagerDelegate() {
+}
+
+ElectronWebModalDialogManagerDelegate::
+    ~ElectronWebModalDialogManagerDelegate() {}
```

Comment:
nit: prefer `= default` here
```suggestion
ElectronWebModalDialogManagerDelegate::ElectronWebModalDialogManagerDelegate() = default;

ElectronWebModalDialogManagerDelegate::
    ~ElectronWebModalDialogManagerDelegate() = default;
```